V4.13.0 (#403)
hCaptcha/hcaptcha-wordpress-plugin: readme.txt

add_filter( 'hcap_activate', 'my_hcap_activate' );
`

> = How to block hCaptcha scripts everywhere except specific page? =

Reviewer: ```suggestion
= How do I block hCaptcha scripts everywhere except on a specific page? =
```

> For instance, to block hCaptcha scripts everywhere except `contact` page, add the following code.

Reviewer: ```suggestion
An an example, to block hCaptcha scripts everywhere except on the `contact` page:
```

add_filter( 'hcap_print_hcaptcha_scripts', 'my_hcap_print_hcaptcha_scripts' );
`

> = How to block hCaptcha styles everywhere except specific page? =

Reviewer: ```suggestion
= How do I block hCaptcha scripts everywhere except on a specific page? =
```

> For instance, to block hCaptcha styles everywhere except `contact` page, add the following code.

Reviewer: ```suggestion
An an example, to block hCaptcha scripts everywhere except on the `contact` page:
```
